pkgs/development/python-modules/rtoml/default.nix

```
@@ -0,0 +1,51 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, setuptools-rust
+, rustc
+, cargo
+, rustPlatform
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "rtoml";
+  version = "0.6.1";
+  # setuptools-rust requires > 2.7
+  disabled = pythonOlder "3.6";
+
+  src = fetchFromGitHub {
+    owner = "samuelcolvin";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "07bf30if1wmbqjp5n4ib43n6frx8ybyxc9fndxncq7aylkrhd7hy";
+  };
+
+  cargoDeps = rustPlatform.fetchCargoTarball {
+    inherit src;
+    name = "${pname}-${version}";
+    sha256 = "1q082sdac5vm4l3b45rfjp4vppp9y9qhagdjqqfdz8gdhm1k8yyy";
+  };
+
+  nativeBuildInputs = with rustPlatform; [
+    setuptools-rust
+    rustc
+    cargo
+    cargoSetupHook
+  ];
+
+  pythonImportsCheck = [ "rtoml" ];
+
+  checkInputs = [ pytestCheckHook ];
+  preCheck = ''
+    cd tests
+  '';
+
+  meta = with lib; {
+    description = "A better TOML library for python implemented in rust";
```

Opinionated change.

```suggestion
    description = "TOML library for Python";
```

"Better", "faster", "super cool", "best in the world" could all be true :wink:

---

this was copied directly from their github page, though i seem to recall capitalizing "Rust"
i agree with your editorial direction but would want to keep a mention of Rust, if for no other reason, that it explains the package name